<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1452</width>
    <height>893</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(68, 68, 68);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1451</width>
     <height>134</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 253, 255);
font-size:28pt;
text-align: center;</string>
   </property>
   <property name="text">
    <string>Link-Saver</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="help">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>800</y>
     <width>1451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(253, 255, 255);
background-color: rgb(162, 162, 121);
font-size:18pt;
text-align: center;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>1451</width>
     <height>671</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>15</number>
    </property>
    <property name="spacing">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_9">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>351</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Math</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/economy-2.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse2">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QFrame" name="frame_7">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>351</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>History</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/photo-1515325595179-59cd5262ca53.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse6">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame_5">
      <property name="styleSheet">
       <string notr="true">border-radius: 20px</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse1">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>100</y>
         <width>351</width>
         <height>221</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>English</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>121</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 20px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/download.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="openExternalLinks">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse1">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QFrame" name="frame_8">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>351</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>World Language</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/download.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse7">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QFrame" name="frame_10">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>351</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>Science</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/images.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse3">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QFrame" name="frame_6">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>351</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;
</string>
       </property>
       <property name="text">
        <string>PE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>111</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-radius: 0px;</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/sports-balls.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse5">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QFrame" name="frame_11">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>351</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Computer Science</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>121</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/Screenshot 2021-06-19 170259.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse4">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QFrame" name="frame_12">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QPushButton" name="launchCourse8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>351</width>
         <height>231</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(206, 197, 146);
font-size:15pt;
border-radius: 20px;</string>
       </property>
       <property name="text">
        <string>Homeroom</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>351</width>
         <height>111</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>../../../../Downloads/Homeroom.jpg</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QToolButton" name="editCourse8">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>10</y>
         <width>25</width>
         <height>19</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: yellow;</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="settings">
   <property name="geometry">
    <rect>
     <x>1350</x>
     <y>30</y>
     <width>75</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;
background-color: gray;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
